torchvision/models/detection/fcos.py

```diff
@@ -201,7 +201,10 @@ def forward(self, x: List[Tensor]) -> Tensor:
 
 class FCOSRegressionHead(nn.Module):
     """
-    A regression head for use in FCOS.
+    A regression head for use in FCOS, which combines regression branch and center-ness branch.
+    This can obtain better performance.
+
+    Reference: `FCOS: A simple and strong anchor-free object detector(updated version) <https://arxiv.org/abs/2006.09214>`_.
```

Comment:
Is the updated version necessary?
```suggestion
    Reference: `FCOS: A simple and strong anchor-free object detector <https://arxiv.org/abs/2006.09214>`_.
```